File: quantum-trading-app/src/main/java/com/quantum/kis/domain/KisEnvironment.java
package com.quantum.kis.domain;

/**
 * KIS API 환경 구분
 * - PROD: 실전투자 환경
 * - VPS: 모의투자 환경
 */
public enum KisEnvironment {
    PROD("실전투자"),
    VPS("모의투자");

    private final String description;

    KisEnvironment(String description) {
        this.description = description;
    }

    public String getDescription() {
        return description;
    }
}

File: quantum-web-api/src/main/kotlin/com/quantum/kis/domain/KisEnvironment.kt
package com.quantum.kis.domain

/**
 * KIS API 환경 정의
 * 
 * @property displayName 표시명
 * @property description 설명
 */
enum class KisEnvironment(
    val displayName: String,
    val description: String
) {
    /**
     * 실전투자 환경 (실제 거래)
     * - REST API: 20건/초
     * - WebSocket: 41개 등록 한도
     */
    LIVE("실전투자", "실제 거래가 가능한 운영 환경"),
    
    /**
     * 모의투자 환경 (테스트 거래)
     * - REST API: 2건/초
     * - WebSocket: 41개 등록 한도
     */
    SANDBOX("모의투자", "테스트 거래를 위한 시뮬레이션 환경")
}